test-jw/src/app.css

```diff
@@ -7,4 +7,8 @@
 
 body {
     height: 200vh;
-}
\ No newline at end of file
+}
+
+.hidden {
+	display: none !important;
```

Comment:
Does it work without `!important` or gets overwritten? 😊

Reply:
The `!important` is crucial for this to work!

Reply:
So, it gets overwritten. But it's a bad practice  😿 Thus why I started looking for other solutions 🤔

Reply:
Hmm, we can avoid setting the _!important_ class by setting the inline style in .jsx component instead. I can give it a go 👍